nav2_lifecycle_manager/src/lifecycle_manager.cpp

```diff
@@ -44,7 +44,9 @@ LifecycleManager::LifecycleManager()
 
   node_names_ = get_parameter("node_names").as_string_array();
   get_parameter("autostart", autostart_);
-  bond_timeout_ = std::chrono::duration<double>(get_parameter("bond_timeout").as_double());
+  int bond_timeout_s;	  
+  get_parameter("bond_timeout", bond_timeout_s);	
+  bond_timeout_ = std::chrono::milliseconds(bond_timeout_s * 1000);
```

Comment:
Please use chrono conversions, not just multiplying by 1000

Reply:
I know you're probably new to all the chrono things, I'm still learning some of them myself. This is a good opportunity to familiarize yourself with it :-)